File: ejercicio_5/SumaPares50Y200.java
public class SumaPares50Y200 {
    public static void main(String[] args) {
        int suma = 0;

        for (int i = 50; i <= 200; i++) {
            if (i % 2 == 0) {
                suma += i;
            }
        }

        System.out.println("Suma total de los números pares entre 50 y 200: " + suma);
    }
}

File: ejercicio_5/suma_pares_50_y_200.py
def main():
    suma = 0

    for i in range(50, 201):
        if i % 2 == 0:
            suma += i

    print(f'Suma total de los números pares entre 50 y 200: {suma}')

if __name__ == '__main__':
    main()
